timetable/views.py

```diff
@@ -292,3 +295,111 @@ def form_valid(self,form):
                 context['rows_failed'] += 1
                 context['errors'].append(row)
         return render(self.request, 'csv_import_success.html', context)
+
+class PrintSubstitutionsView(FormView):
```

Comment:
I think Printing of Substitutions should be for registered users only, but not 100% sure I am not missing something!